geowave-accumulo/pom.xml

```diff
@@ -41,7 +41,7 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
-		<dependency>
+        <dependency>
```

Comment:
this looks to me like it was previously formatted correctly - are you trying to pad your stats? ;)